controllers/categoriesControllers.js

```diff
@@ -0,0 +1,12 @@
+const {selectCategories} = require("../models/categoriesModels");
+
+exports.getCategories = (request, response, next) => {
+    selectCategories()
+    .then((categories) => {
+        response.status(200).send({categories});
+    })
+    .catch((err) => {
+        next(err);
+    });
```

Comment:
This is the correct place to put the catch block, but at the moment it isn't being picked up anywhere? Could we please add some error handling middleware?